importer/urls.py

```diff
@@ -10,5 +10,5 @@
 
 urlpatterns = [
     url(r'^status/$', status, name='status'),
-    url(r'^upload/$', upload, name='upload'),
+    url(r'^upload/(?P<repo_id>\d+)', upload, name='upload'),
```

Comment:
I'm guessing this should have a $ at the end otherwise it may match other suffixes, not sure what Django's behavior is here